src/main/java/com/bwell/auth/model/Client.java

```diff
@@ -0,0 +1,40 @@
+package com.bwell.auth.model;
+
+public class Client {
+    private String url;
+    private String id;
+    private String secret;
+    private String scope;
```

Comment:
can we rename `scope` to `auth_scopes` ?

Reply:
Updated